### ISSUE TITLE
Token networks & channels are not shown for Kovan, production (limits enabled)

### ISSUE THREAD

**Issue by loredanacirstea** [raiden-network/webui#42]
The production version (limits enabled) webui does not show any token networks or channels for the Kovan network.
Even after I created a channel & deposited tokens in it by using the Raiden API directly, the channel did not show up in the webui.

For Mainnet & Ropsten it works correctly.

**Comment by kelsos** [comment]
@loredanacirstea do you have any more information? any console errors? 
Did it fail to display even after refreshing the webui?

**Comment by loredanacirstea** [comment]
I looked at this again and the issue is not from the webui.

- no console errors
- `/api/v1/channels` response is `[]`, even though I have a channel opened: https://kovan.etherscan.io/tx/0x797d9b0e104c59f8cd0c4b168688124836fa4793dec9d0b571fa9ab5375a28b4 . My Raiden node address is `0xbC682A384c48D3bBF6e25Cb10F448F73601B6259`
- `/api/v1/tokens` response is `[]`

Seems like a Raiden client issue, so I have moved the issue here: https://github.com/raiden-network/raiden/issues/3234. Closing this one.
